src/transporters/mqtt.js

```diff
@@ -25,6 +26,10 @@ class MqttTransporter extends Transporter {
 	 * @memberof MqttTransporter
 	 */
 	constructor(opts) {
+		opts = defaultsDeep(opts, {
```

Comment:
It doesn't work if the `opts` is a `String` like `transporter: "mqtt://localhost:1833",`.

Reply:
I've fixed it and changed the option from `qosZero` to `qos`, which is the number and `0` by default.
https://github.com/moleculerjs/moleculer/commit/df3adb5b3f697c9bf949dd1c79114deaad986268

Reply:
Awesome! But I think that if you want the same quality guaranteed by others transport the default QoS should be, moreover I'm not sure that QoS 2 works well with the moleculer logic.